pyquil/operator_estimation.py

```diff
@@ -814,11 +815,71 @@ def serializable(self):
         }
 
 
+def generate_experiment_programs(tomo_experiment: TomographyExperiment,
+                                 active_reset: bool = False) \
+        -> Tuple[List[Program], List[List[int]]]:
```

Comment:
can this fit on the previous line?

Reply:
Not the whole thing. Could put new line at the comma though?

Reply:
bring the ) to this line so you dont have to use the \ maybe?